classes/webservice/WebserviceOutputJSON.php

```diff
@@ -163,6 +163,11 @@ public function renderNodeFooter($node_name, $params)
 
     public function overrideContent($content)
     {
+        array_walk_recursive($this->content, function (&$item) {
+            if (is_null($item)) {
```

Comment:
```suggestion
            if ($item !== null) {
```

Never use is_null function, it's slow :)